### PR TITLE
Adds a new security level: Amber

### PR DESCRIPTION
**Pull request by Tupinambis**
:cl: Tupinambis
add: Adds a new security level between Blue and Red (Amber). The shuttle call time at this level is 8 minutes.
tweak: Blue security level now has a shuttle call time of 12 minutes.
tweak: The security level increase/decrease texts have been modified to accommodate the change.
imageadd: Adds a code Amber sprite for the fire alarms
/:cl:

[why]: There is zero middle ground between blue and red, and I feel like it is detrimental from both an RP and gameplay perspective. Green makes sense, Blue is just an alert level that says "Oh there may or may not be threats. They may or may not be major or minor, who knows.". Red alert can only be triggered if two people with head of staff access swipe (Can be hard to coordinate/not enough heads.), and can sometimes be used for more minor threats, simply because the threat is too much to be blue.

With this change security levels are a bit more meaningful.

Green: No threats
Blue: Potential threats/rumors/inconsequential threats
Amber: There are confirmed active threats that are actively harming crew/the station.
Red: There is an extreme round ending threat to the station
Delta: Station bout to go boom.

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -108,8 +108,11 @@
 						tmp_alertlevel = SEC_LEVEL_GREEN
 					if(tmp_alertlevel < SEC_LEVEL_GREEN)
 						tmp_alertlevel = SEC_LEVEL_GREEN
-					if(tmp_alertlevel > SEC_LEVEL_BLUE)
-						tmp_alertlevel = SEC_LEVEL_BLUE //Cannot engage delta with this
+					if(tmp_alertlevel == SEC_LEVEL_BLUE){
```
Comment (deathride58) [inline code/game/machinery/computer/communications.dm]: Why the brackets

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -108,8 +108,11 @@
 						tmp_alertlevel = SEC_LEVEL_GREEN
 					if(tmp_alertlevel < SEC_LEVEL_GREEN)
 						tmp_alertlevel = SEC_LEVEL_GREEN
-					if(tmp_alertlevel > SEC_LEVEL_BLUE)
-						tmp_alertlevel = SEC_LEVEL_BLUE //Cannot engage delta with this
+					if(tmp_alertlevel == SEC_LEVEL_BLUE){
+						tmp_alertlevel = SEC_LEVEL_BLUE
+						}
```
Comment (deathride58) [inline code/game/machinery/computer/communications.dm]: Why the brackets

File: code/game/machinery/computer/communications.dm [hunk 3]
```diff
@@ -390,8 +393,11 @@
 				tmp_alertlevel = SEC_LEVEL_GREEN
 			if(tmp_alertlevel < SEC_LEVEL_GREEN)
 				tmp_alertlevel = SEC_LEVEL_GREEN
-			if(tmp_alertlevel > SEC_LEVEL_BLUE)
-				tmp_alertlevel = SEC_LEVEL_BLUE //Cannot engage delta with this
+			if(tmp_alertlevel == SEC_LEVEL_BLUE){
```
Comment (deathride58) [inline code/game/machinery/computer/communications.dm]: Why the brackets

File: code/game/machinery/computer/communications.dm [hunk 4]
```diff
@@ -390,8 +393,11 @@
 				tmp_alertlevel = SEC_LEVEL_GREEN
 			if(tmp_alertlevel < SEC_LEVEL_GREEN)
 				tmp_alertlevel = SEC_LEVEL_GREEN
-			if(tmp_alertlevel > SEC_LEVEL_BLUE)
-				tmp_alertlevel = SEC_LEVEL_BLUE //Cannot engage delta with this
+			if(tmp_alertlevel == SEC_LEVEL_BLUE){
+				tmp_alertlevel = SEC_LEVEL_BLUE
+				}
```
Comment (deathride58) [inline code/game/machinery/computer/communications.dm]: Why the brackets

**Review by deathride58 — DISMISSED** [review]
This isn't C

**Comment by Ragolution** [comment]
Isn't amber alert "lost child"?

**Comment by silicons** [comment]
*yawn

**Review by silicons — APPROVED** [review]

**Pull request merged**